Trap Quest.inform/Inform/Extensions/Every Turn/Periodic Effects.i7x

```diff
@@ -207,6 +207,54 @@ To compute stat healing:
 		decrease temp_int_dam by 1;
 		say "[bold type]You feel as though your mind is slightly clearer![roman type]";
 
+A time based rule (this is the audible jiggling rule):
+	if a random number between 1 and 100 < 50 + unlucky:
+		if debugmode > 0, say "AUDIBLE CHECK. ";
+		let audibly jiggling clothing be nothing;
+		repeat with C running through worn cursed blandness clothing:
+			if C is overdress or C is bikini top or C is bra:
+				If C is chestless or fully exposing, decide on 0;
+				If the largeness of breasts <= 2,decide on 0;
+				if C is overdress and support of (C - a bra) is >= 2, decide on 0;
+				if C is plentiful or rare and a random number between 1 and 3 is 2, now audible jiggling clothing is C;
+		repeat with C running through worn audible jiggling clothing:
+			now audibly jiggling clothing is C;
+		
+		To decide which number is the stealth influence of (C - a clothing)
+			if C is bsounding, decide on -6;
+			decide on 0;
+		
+		To compute bsound:
+			repeat with C running through audibly jiggling clothing:
+				say "Your [ShortDesc of C] accompanies the bounces of your [ShortDescBreasts] with soft boings and wobbles.[or] Your [C] is ensuring that every [BreastBounceDesc] your [RealBreastDesc]. [or]You are unable to prevent your [C] from letting everyone know each bounce of your [ShortDescBreasts]![or][bold type]The jiggling of your [ShortDescBreasts] is likely to attract others to your location![roman type][line break][or] The noises your [ShortDesc of C] is making signals exactly where you are .[or]You wish you could stop your [C] from detailing your breasts as they [BreastBounceDesc] .[as decreasingly likely outcomes]";
```

Comment:
Again, the correct terms are either `[ShortDesc of breasts]` or `[BreastDesc]`.

Furthermore there's a couple of times here where you have put whitespace before the `.` instead of after, at the end of the last two segments of your `[one of]` construction.

There's also an unnecessary whitespace before "The noises".